docker-compose.yml

```diff
@@ -188,25 +188,24 @@ services:
     volumes:
       - trino-postgres:/data
 
-  hive-metastore-mariadb:
-    image: mariadb:10.11.5
+  hive-metastore-db:
+    image: postgres:16.0-alpine3.18
     environment:
-      MYSQL_ALLOW_EMPTY_PASSWORD: "true"
-      MYSQL_USER: admin
-      MYSQL_PASSWORD: admin
-      MYSQL_DATABASE: metastore_db
+      POSTGRES_USER: admin
+      POSTGRES_PASSWORD: admin
+      POSTGRES_DB: metastore
     healthcheck:
       interval: 1s
       retries: 30
       test:
         - CMD
-        - mariadb-admin
-        - ping
+        - pg_isready
       timeout: 5s
+    command: -c port=23456
```

Comment:
Have to use a different port than the default of 5432, which is bound by the `trino-postgres` container.